static/src/js/app/constants.js

```diff
@@ -27,3 +31,8 @@ export const RATINGS = {
   4: 'I definitely know what this word means but could forget soon',
   5: 'I know this word so well, I am unlikely to ever forget it',
 };
+
+export const LANGUAGES = {
+  lat: 'Latin',
+  grc: 'Greek',
```

Comment:
add russian

Reply:
@Tanvez Can you add `rus: 'Russian'` to the languages for now, so that it matches the `SUPPORTED_LANGUAGES` in `hedera/settings.py`? Eventually we can add an endpoint for that, but just want to make sure it matches for now.

Reply:
Nevermind, I just missed the fact that you already added this. My bad!